azure-amqp/cpp/cpp.json

```diff
@@ -1,5 +1,5 @@
 {
-  "ldflags": "-liothub_client -liothub_client_amqp_transport -laziotsharedutil -luamqp -lpthread -lcurl -lssl -lcrypto -luuid -lmraa -lupm-grove",
-  "cxxflags": "-std=c++11 -I/usr/include/azureiot -I/usr/include/azureiot/inc",
+  "ldflags": "-L ../src -liothub_client -liothub_client_amqp_transport -laziotsharedutil -luamqp -lpthread -lcurl -lssl -lcrypto -luuid -lmraa -lupm-grove",
+  "cxxflags": "-std=c++11 -I/usr/include/azureiot -I/usr/include/azureiot/inc -L ../src",
```

Comment:
Do you need to add "-L ../src" to the cxx flags as well? Wouldn't it be enough to add to ldflags?

Reply:
I verified that only ldflags are needed.